mlfinlab/online_portfolio_selection/__init__.py

```diff
@@ -23,10 +23,14 @@
 from mlfinlab.online_portfolio_selection.mean_reversion.online_moving_average_reversion import OnlineMovingAverageReversion
 
 # Pattern Matching
-# from mlfinlab.online_portfolio_selection.pattern_matching.correlation_driven_nonparametric_learning import CorrelationDrivenNonparametricLearning
-# from mlfinlab.online_portfolio_selection.pattern_matching.CORN_U import CORN_U
-# from mlfinlab.online_portfolio_selection.pattern_matching.CORN_K import CORN_K
-# from mlfinlab.online_portfolio_selection.pattern_matching.SCORN import SCORN
-# from mlfinlab.online_portfolio_selection.pattern_matching.SCORN_K import SCORN_K
-# from mlfinlab.online_portfolio_selection.pattern_matching.FCORN import FCORN
-# from mlfinlab.online_portfolio_selection.pattern_matching.FCORN_K import FCORN_K
+from mlfinlab.online_portfolio_selection.pattern_matching.correlation_driven_nonparametric_learning import CorrelationDrivenNonparametricLearning
+from mlfinlab.online_portfolio_selection.pattern_matching.correlation_driven_nonparametric_learning_uniform import CorrelationDrivenNonparametricLearningUniform
+from mlfinlab.online_portfolio_selection.pattern_matching.correlation_driven_nonparametric_learning_k import CorrelationDrivenNonparametricLearningK
+from mlfinlab.online_portfolio_selection.pattern_matching.symmetric_correlation_driven_nonparametric_learning import \
```

Comment:
@Jackal08 actually mentioned this before saying that it might be better to keep the whole strategy for interpretability, but I do agree the name length is getting out of hands.